packages/wdio-browserstack-service/src/util.ts

```diff
@@ -526,7 +588,66 @@ export const getA11yResults = async (browser: WebdriverIO.Browser, isBrowserStac
     }
 }
 
-export const getA11yResultsSummary = async (browser: WebdriverIO.Browser, isBrowserStackSession?: boolean, isAccessibility?: boolean | string) : Promise<{ [key: string]: any; }> => {
+export const getAppA11yResults = async (isAppAutomate: boolean, browser: WebdriverIO.Browser, isBrowserStackSession?: boolean, isAccessibility?: boolean | string, sessionId?: string | null) : Promise<Array<{ [key: string]: any; }>> => {
+    if (!isBrowserStackSession) {
+        return [] // since we are running only on Automate as of now
+    }
+
+    if (!isAppAccessibilityAutomationSession(isAccessibility, isAppAutomate)) {
+        BStackLogger.warn('Not an Accessibility Automation session, cannot retrieve Accessibility results summary.')
+        return []
+    }
+
+    try {
+        BStackLogger.debug('Performing scan before getting results summary')
+        await performA11yScan(isAppAutomate, browser, isBrowserStackSession, isAccessibility)
+        const apiUrl = `${APP_ALLY_ENDPOINT}/${APP_ALLY_ISSUES_ENDPOINT}`
+        let upperTimeLimit = Date.now() + 360000 // 30 seconds
+        if (process.env[BSTACK_A11Y_POLLING_TIMEOUT]) {
+            upperTimeLimit = Date.now() + parseInt(process.env[BSTACK_A11Y_POLLING_TIMEOUT])
+        }
+        const params = { test_run_uuid: process.env.TEST_ANALYTICS_ID, session_id: sessionId, timestamp: Date.now() } // Query params to pass
+        const header = { Authorization: `Bearer ${process.env.BSTACK_A11Y_JWT}` }
+        const apiRespone = await pollApi(apiUrl, params, header, upperTimeLimit)
+        BStackLogger.debug(`apiRespone: ${JSON.stringify(apiRespone)}`)
+        const result = apiRespone?.data?.data?.issues
+        BStackLogger.debug(`Polling Result: ${JSON.stringify(result)}`)
+        return result
+    } catch {
+        BStackLogger.error('No accessibility summary was found.')
+        return []
+    }
+}
+
+export const getAppA11yResultsSummary = async (isAppAutomate: boolean, browser: WebdriverIO.Browser, isBrowserStackSession?: boolean, isAccessibility?: boolean | string, sessionId?: string | null) : Promise<{ [key: string]: any; }> => {
+    if (!isBrowserStackSession) {
+        return {} // since we are running only on Automate as of now
+    }
+
+    if (!isAppAccessibilityAutomationSession(isAccessibility, isAppAutomate)) {
+        BStackLogger.warn('Not an Accessibility Automation session, cannot retrieve Accessibility results summary.')
+        return {}
+    }
+
+    try {
+        BStackLogger.debug('Performing scan before getting results summary')
+        await performA11yScan(isAppAutomate, browser, isBrowserStackSession, isAccessibility)
+        const apiUrl = `${APP_ALLY_ENDPOINT}/${APP_ALLY_ISSUES_SUMMARY_ENDPOINT}`
+        const upperTimeLimit = Date.now() + 360000 // 30 seconds
```

Comment:
360000 ms = 360 secs right? not 30 secs

Reply:
We have changed it to 30 secs as default